### PR TITLE
DOCS-10761 made edits

### PR DESCRIPTION
**Pull request by priyaayyar**

### CODE REVIEW

**Review by sbehrends — DISMISSED** [review]

File: modules/ROOT/pages/manage-exchange-api-task.adoc [hunk 1]
```diff
@@ -34,10 +34,16 @@ Use this option if you want to provide additional security for your API.
 +
 For example, you can use the URL of your API asset in Exchange. In this case, the API is consuming the JSONPlaceholder API, so enter `+http://jsonplaceholder.typicode.com+`.
 +
-. If you are creating an endpoint with proxy, accept the default path `/` for the proxy endpoint in the *Path* field, or enter another path. 
+. If you are creating an endpoint with proxy, accept the base path `/` for the proxy endpoint in the *Path* field, or enter another path. 
 +
-For example, changing the path to `/users` makes the REST call to the service at the implementation URI `+http://jsonplaceholder.typicode.com/users+`.
+For example, if an API is defined with the `Implementation URI` set to `http://jsonplaceholder.typicode.com` the proxy returns the following URIs based on the value you configure in the proxy path:
```
Comment (Cristian-Venticinque) [inline modules/ROOT/pages/manage-exchange-api-task.adoc]: `http://jsonplaceholder.typicode.com` the proxy > `http://jsonplaceholder.typicode.com`**,** the proxy

File: modules/ROOT/pages/manage-exchange-api-task.adoc [hunk 2]
```diff
@@ -34,10 +34,16 @@ Use this option if you want to provide additional security for your API.
...
 +
+[%header%autowidth.spread,cols="a,a,a"]
+|===
+| *Proxy Path* | *Request* | *Returned by Proxy*
```
Comment (Cristian-Venticinque) [inline modules/ROOT/pages/manage-exchange-api-task.adoc]: Suggestion: *Returned by Proxy* > **Response from Proxy** or just **Response**

**Review by Cristian-Venticinque — DISMISSED** [review]

File: modules/ROOT/pages/manage-exchange-api-task.adoc (hunk 2 above)
Comment (priyaayyar) [reply modules/ROOT/pages/manage-exchange-api-task.adoc]: Done! Thanks!

File: modules/ROOT/pages/manage-exchange-api-task.adoc (hunk 1 above)
Comment (priyaayyar) [reply modules/ROOT/pages/manage-exchange-api-task.adoc]: Good catch! Missed that. THanks!

**Review by Cristian-Venticinque — APPROVED** [review]

**Pull request merged**